iceoryx_posh/cmake/iceoryx_versions.hpp.in

```diff
@@ -1,8 +1,14 @@
 #ifndef __ICEORYXVERSIONS__
 #define __ICEORYXVERSIONS__
 
+#define ICEORYX_VERSION_MAJOR    @PROJECT_VERSION_MAJOR@
+#define ICEORYX_VERSION_MINOR    @PROJECT_VERSION_MINOR@
+#define ICEORYX_VERSION_PATCH    @PROJECT_VERSION_PATCH@
+#define ICEORYX_VERSION_TWEAK    @PROJECT_VERSION_TWEAK@
```

Comment:
@elBoberido @dkroenke @budrus Do we really need such a fine granularity with the versions? I completely accept major, minor and patch. But do we really need a tweak version?

Or is tweak something like 0.17.0-rc1 for release candidate?

Reply:
yes, tweak is used for such things as RCs

Reply:
I think we should keep it then as we're using RCs quite often.